backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -229,6 +229,66 @@ class TestOperator extends WholeStageTransformerSuite {
     assert(df.queryExecution.executedPlan.find(_.isInstanceOf[ProjectExecTransformer]).isDefined)
   }
 
+  test("Test ceil function") {
+    val df = spark.sql("SELECT ceil(cast(l_orderkey as long)) from lineitem limit 1")
+    val result = df.collect()
+    assert(result.length == 1)
+    val expected = Seq(Row(1))
+    TestUtils.compareAnswers(result, expected)
```

Comment:
Seems to be better if using `runQueryAndCompare`. It can be referred in: https://github.com/oap-project/gluten/pull/451/files#diff-d6f3816bd1517d3d8708ba5673298038b807726d79abf2b8f8f0584aac636752.

Reply:
Because, also need to check  DataFrame plan. So didn't use runQueryAndCompare directly.

Reply:
Where is the DataFrame plan checked for now?

Reply:
OK, got it.